examples/pybullet/unittests/userDataTest.py

```diff
@@ -0,0 +1,239 @@
+import unittest
+import pybullet
+
+from pybullet_utils import bullet_client
+
+PLANE_PATH = "plane.urdf"
+ROBOT_PATH = "r2d2.urdf"
+
+
+class TestUserDataMethods(unittest.TestCase):
+
+	@classmethod
+	def setUpClass(cls):
+		TestUserDataMethods.server = bullet_client.BulletClient(connection_mode=pybullet.GUI_SERVER)
```

Comment:
SHARED_MEMORY_SERVER to allow running with GUI/headless

Reply:
Done.